inc/ProgEvent.h

```diff
@@ -126,6 +124,10 @@ namespace PCOE {
          *  @see        ProgEvents
          **/
         void setNPoints(const unsigned int nPoints);
+
+    private:
+        unsigned int nSamples; ///< Number of samples in the occurrence matrix
```

Comment:
I Strongly prefer `sampleCount` rather than `nSamples` since it mirrors the terminology used in most standard libraries. Since we are already moving this, can we go ahead and rename it at the same time?

@teubert Is there any specific reason to keep the "nThing" terminology?

Reply:
I agree, sampleCount is a better name.